doc/en/getting-started.rst

```diff
@@ -19,17 +19,15 @@ Installation and Getting Started
 Installation
 ----------------------------------------
 
-Installation options::
+Installation::
 
-    pip install -U pytest # or
-    easy_install -U pytest
+    pip install -U pytest
 
 To check your installation has installed the correct version::
 
     $ pytest --version
     This is pytest version 3.0.0, imported from $PYTHON_PREFIX/lib/python3.5/site-packages/pytest.py
 
-If you get an error checkout :ref:`installation issues`.
```

Comment:
It doesn't... I didn't really track where it might have went, but I remember there being something related to weird Windows installation problems because "python" wasn't on the PATH... don't think it is really relevant if that's what it was.